src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -27,6 +27,11 @@
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeHandleMinusOneIsInvalid.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeWaitHandle.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Condition="'$(FeatureWin32Registry)' == 'true'" Include="$(MSBuildThisFileDirectory)\Microsoft\Win32\SafeHandles\SafeRegistryHandle.cs" />
```

Comment:
This will either need to be in the regular `Condition="'$(TargetsWindows)'=='true' and '$(EnableWinRT)'!='true'"` block (I think it is what I would prefer), or FeatureWin32Registry needs to be defined somewhere in CoreRT.